site/index.html

```diff
@@ -5,29 +5,48 @@
             </div>
          </div>
          <div class=" span6 col-md-6">
-            <div class="branding sample">
-               <br class="visible-xs visible-sm">
-               <h4 class="tag">Declarative, statically typed code.</h4>
-               <div title="This example is contrived in order to demonstrate what Haskell looks like, including: (1) where syntax, (2) enumeration syntax, (3) pattern matching, (4) consing as an operator, (5) list comprehensions, (6) infix functions. Don&#39;t take it seriously as an efficient prime number generator." class="code-sample">
-                  <pre><span class='hs-definition'>primes</span> <span class='hs-keyglyph'>=</span> <span class='hs-varid'>filterPrime</span> <span class='hs-keyglyph'>[</span><span class='hs-num'>2</span><span class='hs-keyglyph'>..</span><span class='hs-keyglyph'>]</span>
+            <div class="row" id="code-example">
+               <div class="branding sample">
+                  <br class="visible-xs visible-sm">
+                  <h4 class="tag">Declarative, statically typed code.</h4>
+                  <div title="This example is contrived in order to demonstrate what Haskell looks like, including: (1) where syntax, (2) enumeration syntax, (3) pattern matching, (4) consing as an operator, (5) list comprehensions, (6) infix functions. Don&#39;t take it seriously as an efficient prime number generator." class="code-sample">
+                     <pre><span class='hs-definition'>primes</span> <span class='hs-keyglyph'>=</span> <span class='hs-varid'>filterPrime</span> <span class='hs-keyglyph'>[</span><span class='hs-num'>2</span><span class='hs-keyglyph'>..</span><span class='hs-keyglyph'>]</span>
   <span class='hs-keyword'>where</span> <span class='hs-varid'>filterPrime</span> <span class='hs-layout'>(</span><span class='hs-varid'>p</span><span class='hs-conop'>:</span><span class='hs-varid'>xs</span><span class='hs-layout'>)</span> <span class='hs-keyglyph'>=</span>
           <span class='hs-varid'>p</span> <span class='hs-conop'>:</span> <span class='hs-varid'>filterPrime</span> <span class='hs-keyglyph'>[</span><span class='hs-varid'>x</span> <span class='hs-keyglyph'>|</span> <span class='hs-varid'>x</span> <span class='hs-keyglyph'>&lt;-</span> <span class='hs-varid'>xs</span><span class='hs-layout'>,</span> <span class='hs-varid'>x</span> <span class='hs-varop'>`mod`</span> <span class='hs-varid'>p</span> <span class='hs-varop'>/=</span> <span class='hs-num'>0</span><span class='hs-keyglyph'>]</span></pre>
+                  </div>
+               </div>
+            </div>
+            <div class="row" id="get-started-button">
+               <div class="span12 col-sm-12" style="padding-top: 10px;">
+                  <br/>
+                  <p class="text-center">
+                     <a href="/get-started/" class="btn btn-lg btn-primary" style="font-size: 1.5em">GET STARTED</a>
```

Comment:
```suggestion
                     <a href="/get-started/" class="btn btn-lg btn-primary" style="font-size: 1.5em">Get Started</a>
```
I would use the same color as the purple background in the section below it, and I think the Haskell logo is now no aligned with the right side, but I'm not expert in this.

Reply:
Before it was aligned regarding the top border, now it is aligned vertically, due to the space that the button occupies. That said, I am no designer so I can't tell what is better.